java/serving/src/test/java/feast/serving/it/ServingRedisAzureRegistryIT.java

```diff
@@ -50,8 +50,10 @@ private static BlobServiceClient createClient() {
     return new BlobServiceClientBuilder()
         .endpoint(
             String.format(
-                "http://localhost:%d/%s",
-                azureBlobMock.getMappedPort(BLOB_STORAGE_PORT), TEST_ACCOUNT_NAME))
+                "http://%s:%d/%s",
```

Comment:
nit: f-strings are a little nicer in many ways.

Reply:
😄 this is java, i don't think there's a better option

Reply:
🤦‍♂️ this is what I get for reviewing PRs before coffee